great_expectations_cloud/agent/actions/generate_data_quality_check_expectations_action.py

```diff
@@ -131,16 +148,34 @@ def _get_metrics(self, data_asset: DataAsset) -> tuple[MetricRun, UUID]:
 
         return metric_run, metric_run_id
 
+    def _add_volume_change_expectation(self, asset_id: UUID) -> UUID:
+        expectation = gx_expectations.ExpectTableRowCountToBeBetween(
+            windows=[
+                Window(
+                    constraint_fn="mean",
+                    parameter_name="min_value",
+                    range=1,
+                    offset=Offset(positive=0.0, negative=0.0),
+                )
```

Comment:
My `strict` work introduced `strict` to `Window`. You will need to pass `strict=True` here to get the correct config/rendering. This is probably different than what was given in the ticket, but my work changed the config.